apps/mosaico/src/Mosaico/Header.purs

```diff
@@ -108,7 +108,13 @@ render { state: { menuVisible, menuComponent }, props } =
                  else
                    [ searchButton ])
                 <> [ DOM.div_
-                      [ menuComponent { visible: menuVisible }
+                      [ menuComponent
```

Comment:
I think we can get rid of this instance of the menu component here, as this is a leftover from back when the menu was still an overlay.

Reply:
I see 🤔 I'll do a cleanup

Reply:
I guess all the `menuVisible` stuff as well